### DISCUSSION TITLE
New student plan

### DISCUSSION THREAD

**Discussion by EwoutH** [codota/TabNine#589] [General]
Until mid-2022 there was a student plan available for TabNine, but unfortunately it currently isn't available anymore. Are there any plans to introduce a new student plan for TabNine?

![Screenshot_80](https://user-images.githubusercontent.com/15776622/220434210-e060b9ce-76bd-4652-883f-004818a8677f.png)

**Comment by FahimFBA** [comment]
I don't think they have any plans to provide free support for the students anymore. Their price is already more than GitHub copilot.

**Reply by sascharo** [reply]
And GitHub Copilot is free for students. By the way, AWS Q as well.
